plugins/coverimage.koplugin/main.lua

```diff
@@ -176,7 +175,7 @@ function CoverImage:addToMainMenu(menu_items)
             -- menu entry: set fallback image
             {
                 text_func = function()
-                    return self.cover_image_path and _("Set fallback image when no cover")
+                    return self.cover_image_path and _("Set fallback image when no cover or excl. book")
```

Comment:
This text_func would return `nil` when self.cover_image_path is false.
I guess you can just let it be `text = _("Set fallback image path")` and add a `help_text = _("...")` with mode detailed info and advices (shown on long-press).

Reply:
And yes, for such a specific plugin, you can add a 1st menu item with detailed info (like we have in `Typography rules> About...`)